ruby-gem/Rakefile

```diff
@@ -36,14 +35,31 @@ def build
 end
 
 task :build do
```

Comment:
The reason I am recommending this because in CI, you want to be able to build the gem:

```
$ rake build
rake aborted!
LoadError: cannot load such file -- calabash-android/java_keystore
/Users/moody/git/calabash/calabash-android/ruby-gem/lib/calabash-android/helpers.rb:6:in `<top (required)>'
lib/calabash-android/env.rb:1:in `require_relative'
lib/calabash-android/env.rb:1:in `<top (required)>'
/Users/moody/git/calabash/calabash-android/ruby-gem/Rakefile:2:in `load'
/Users/moody/git/calabash/calabash-android/ruby-gem/Rakefile:2:in `<top (required)>'
(See full trace by running task with --trace)
```

I realize the TestServer.apk will be missing, but since the .apk is not required for unit testing, I'd like to do something like this:

```
$ touch lib/calabash-android/lib/TestServer.apk
$ rake build
```

to confirm that the gem _will build_.

The Calabash iOS gem has some nice checks to make sure that you don't release without the required files:

* https://github.com/calabash/calabash-ios/blob/develop/calabash-cucumber/Rakefile#L26